lms/templates/header/header.html

```diff
@@ -77,6 +77,10 @@
     </div>
     <div class="mobile-menu hidden" aria-label=${_("More Options")} role="menu" id="mobile-menu"></div>
     % if user.is_authenticated:
+        <% 
+            colaraz_profile = ColarazUserProfile.objects.get(user=user) 
+            prefix = colaraz_profile.site_identifier
```

Comment:
Can't we get the profile object using `user.profile`?

Reply:
done